Sources/Networking/Operations/PostReceiptDataOperation.swift

```diff
@@ -188,6 +192,7 @@ extension PostReceiptDataOperation.PostData: Encodable {
         case attributes
         case aadAttributionToken
         case presentedOfferingIdentifier
+        case testReceiptIdentifier = "receipt_id"
```

Comment:
@antoniobg new parameter name. Does this look good?

Reply:
It does!